ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master.yaml

```diff
@@ -28,8 +28,8 @@ images:
       - registry.svc.ci.openshift.org/openshift/release:golang-1.15
   to: assisted-installer-agent
 promotion:
-  name: "4.8"
-  namespace: ocp
+  name: assisted
+  namespace: ocp-kni
```

Comment:
Please also delete the (now non-matching) configs for `release-4.x` branches that are still configured to promote to `ocp`

Reply:
...in all repos' config